src/components/wallet/views/claimableDisplay.tsx

```diff
@@ -0,0 +1,58 @@
+import { h } from '@stencil/core'
+import { has as loHas } from 'lodash-es'
+import WalletButton from './walletButton'
+import { Wallet } from '../wallet'
+
+interface ClaimableBalance {
+  id: string
+  asset: string
+  amount: string
+  sponsor?: string
+  claimants: any[]
+}
+
+export default function claimableDisplay(this: Wallet) {
+  if (this.account.availableBalances) {
+    const claimableRecords = (claimableBalances: any) => {
```

Comment:
No worries if not but are we able to use a proper type or no? Is an item in `claimableBalances` a `claimableBalanceRecord`?